nucliadb/nucliadb/search/search/merge.py

```diff
@@ -355,17 +397,20 @@ async def merge_results(
     show: List[ResourceProperties],
     field_type_filter: List[FieldTypeName],
     extracted: List[ExtractedDataTypeName],
+    query: SearchRequest,
```

Comment:
NIT: would it make more sense to pass the relations metadata of the query?(ie: `query.relations` for instance)